inttest/footloose-alpine/Dockerfile

```diff
@@ -26,3 +26,10 @@ ENV KUBECONFIG=/var/lib/k0s/pki/admin.conf
 # Install etcd for smoke tests with external etcd
 RUN curl -L https://github.com/etcd-io/etcd/releases/download/v$ETCD_VERSION/etcd-v$ETCD_VERSION-linux-$ETCD_ARCH.tar.gz \
   | tar xz -C /opt --strip-components=1
+
+# This lets etcd start when running on arm in smokes
+ENV ETCD_UNSUPPORTED_ARCH=arm64
```

Comment:
didn't the etcd start supporting arm64 natively?

Reply:
xref #1624

Reply:
I guess this comes from some rebase gone wild or something, will remove